source/_static/css/widget-sidebar.css

```diff
@@ -0,0 +1,143 @@
+.rst-widget-sidebar::-webkit-scrollbar {
+  /* This is the magic bit */
```

Comment:
Did you write this? If so could you write a more descriptive comment for future archaeologists? :)

Reply:
I used the css declarations from the RTD theme and mass replaced `rst-versions` with `rst-widget-sidebar`. The line you referenced was copied from from StackOverflow to hide scrollbars in webkit/webengine browsers. I can add the link.